webpack/assets/javascripts/react_app/redux/API/APIHelpers.js

```diff
@@ -0,0 +1,19 @@
+import { API_OPERATIONS } from './APIConstants';
+import { API } from './';
+
+export const getApiMethodByActionType = type => type.substring(4).toLowerCase();
```

Comment:
This feels a bit hacky, it can be `split('API_')[1].toLowerCase()` so it will be more clear
or even `Object.keys(API_OPERATIONS).find(key => object[key] === type).toLowerCase();`

Reply:
it was requested to change by @amirfefer in: https://github.com/theforeman/foreman/pull/7261#discussion_r366440995